MobileWallet/Screens/Debug/UIViewControllerDebugExtension.swift

```diff
@@ -199,7 +199,7 @@ extension UIViewController: MFMailComposeViewControllerDelegate {
         return archiveURL
     }
 
-    private func onSendFeedback() {
+    func onSendFeedback() {
```

Comment:
I think maybe move this function to the settings VC. It's not a debug feature anymore.

Reply:
this function is used for report bug after shake